### PR TITLE
Followup to exception refactoring

### PR DESCRIPTION
**Pull request by dannon**
Fixes a few issues @nsoranzo noticed with my semiautomatic refactorings.

### CODE REVIEW

File: lib/galaxy/webapps/galaxy/api/lda_datasets.py [hunk 1]
```diff
@@ -607,7 +607,7 @@ def traverse( folder ):
                 log.exception( "Unable to create archive for download" )
                 raise exceptions.InternalServerError( "Unable to create archive for download." )
             except Exception:
-                log.exception( "Unexpected error %s in create archive for download", sys.exc_info()[ 0 ] )
+                log.exception( "Unexpected error %s in create archive for download" )
```
Comment (nsoranzo) [inline lib/galaxy/webapps/galaxy/api/lda_datasets.py]: Need to remove ` %s` too.

File: lib/galaxy/webapps/galaxy/controllers/library_common.py [hunk 2]
```diff
@@ -1863,7 +1863,7 @@ def __str__( self ):
                     status = 'error'
                 except:
                     error = True
-                    log.exception( "Unexpected error %s in create archive for download", sys.exc_info()[0] )
+                    log.exception( "Unexpected error %s in create archive for download" )
```
Comment (nsoranzo) [inline lib/galaxy/webapps/galaxy/controllers/library_common.py]: Same here.

**Review by nsoranzo — CHANGES_REQUESTED** [review]

File: lib/galaxy/webapps/galaxy/api/lda_datasets.py (hunk 1 above)
Comment (dannon) [reply lib/galaxy/webapps/galaxy/api/lda_datasets.py]: Yeah, one more commit coming.  sys is also unused now.

**Comment by dannon** [comment]
xref #3933

**Review by nsoranzo — APPROVED** [review]

**Pull request merged**